services/apps/src/apps/service/workspace.clj

```diff
@@ -1,9 +1,11 @@
 (ns apps.service.workspace
   (:use [korma.db :only [transaction]])
-  (:require [apps.persistence.workspace :as wp]))
+  (:require [apps.clients.iplant-groups :as iplant-groups]
+            [apps.persistence.workspace :as wp]))
 
 (defn get-workspace
-  [{:keys [username]}]
+  [{short-username :shortUsername :keys [username]}]
+  (iplant-groups/add-de-user short-username)
```

Comment:
Ah, but this is the main function of the `workspaces` endpoint, and the `bootstrap` endpoint is in `terrain`.
Does `terrain` have an `iplant-groups` client? Or is that all contained in the `apps` service?